### PR TITLE
Added new cmdlets for Diagnostics

### PR DESCRIPTION
**Pull request by SiddharthChatrolaMs**
Fixing powershell documentation
Upgrade to latest azure redis management package
Cmdlets for enabling and disabling Diagnostics 
Warning message in all managed cache commands

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@SiddharthChatrolaMs__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (sichatro). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by markcowl** [comment]
@SiddharthChatrolaMs There are merge conflicts, please update

File: src/ResourceManager/RedisCache/Commands.RedisCache.Test/Commands.RedisCache.Test.csproj [hunk 1]
```diff
@@ -56,8 +56,12 @@
     <Reference Include="Microsoft.Azure.Common.NetFramework">
       <HintPath>..\..\..\packages\Microsoft.Azure.Common.2.1.0\lib\net45\Microsoft.Azure.Common.NetFramework.dll</HintPath>
     </Reference>
-    <Reference Include="Microsoft.Azure.Management.Redis, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Redis.1.1.0-preview\lib\net40\Microsoft.Azure.Management.Redis.dll</HintPath>
+    <Reference Include="Microsoft.Azure.Insights, Version=0.9.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Microsoft.Azure.Insights.0.9.1-preview\lib\net45\Microsoft.Azure.Insights.dll</HintPath>
```
Comment (markcowl) [inline src/ResourceManager/RedisCache/Commands.RedisCache.Test/Commands.RedisCache.Test.csproj]: Make sure this is updated for all libraries using Insights.

File: src/ResourceManager/RedisCache/Commands.RedisCache.Test/Commands.RedisCache.Test.csproj [hunk 2]
```diff
@@ -72,6 +76,18 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\..\packages\Microsoft.Azure.Test.HttpRecorder.1.0.5799.28345-prerelease\lib\net45\Microsoft.Azure.Test.HttpRecorder.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.Data.Edm, Version=5.6.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```
Comment (markcowl) [inline src/ResourceManager/RedisCache/Commands.RedisCache.Test/Commands.RedisCache.Test.csproj]: These new files will require a wix file update

**Comment by markcowl** [comment]
@SiddharthChatrolaMs Also, please update the module version in AzureRm.RedisCache.psd1

**Comment by SiddharthChatrolaMs** [comment]
@markcowl  Fixed wix file. (When generated wix it contained lot of changes in non-redis projects, I have reverted them and just pushed redis related fix)

**Pull request merged**